### PR TITLE
Copy packages is no longer needed in cos package

### PR DESCRIPTION
**Pull request by davidcassany**
This is a follow up PR from #906

Signed-off-by: David Cassany <dcassany@suse.com>

### CODE REVIEW

**Review by mudler — APPROVED** [review]
Nais!  :partying_face:

**Review by mudler — APPROVED** [review]
Looks good here, nice cleanup !

File: packages/toolchain/collection.yaml [hunk 1]
```diff
@@ -21,22 +21,32 @@ packages:
...
     upx: false
     fips: false
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
+      category: "toolchain"
+      # Would be nice to restrict to the luet version defined within this file
```
Comment (mudler) [inline packages/toolchain/collection.yaml:34]: can use YAML anchors for that: define luet once, and refer it in multiple places, like we do in the cos collection

File: packages/toolchain/collection.yaml [hunk 2]
```diff
@@ -21,22 +21,32 @@ packages:
...
-    version: 0.3.8-15
+    version: 0.3.8-16
     upx: false
     fips: false
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
```
Comment (mudler) [inline packages/toolchain/collection.yaml:32]:
```suggestion
    - name: "luet" &luet
```
Not tested :sweat_smile:

File: packages/toolchain/collection.yaml [hunk 3]
```diff
@@ -21,22 +21,32 @@ packages:
...
-    version: 0.3.8-16
+    version: 0.3.8-17
     upx: false
     fips: true
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
```
Comment (mudler) [inline packages/toolchain/collection.yaml:46]:
```suggestion
    - <<: *luet
```

Comment (davidcassany) [reply packages/toolchain/collection.yaml:46]: I saw that in `cos` package, but honestly I did not understand it and I did not want to mess with something I do not understand. Where is this logic taking place? Is this part of luet? Part of the yaml parser? Shouldn't this line `<<: *luet` replace the whole object (I mean `name`, `category`, `version`...)?

Comment (davidcassany) [reply packages/toolchain/collection.yaml:46]: Oh cool, I'll give it a try

**Pull request merged**